### ISSUE TITLE
Debug panel bug with the new vscode version

### ISSUE THREAD

**Issue by lincolnaleixo** [iocave/customize-ui#36]
![Screen Shot 2020-03-17 at 1 56 40 AM](https://user-images.githubusercontent.com/2906581/76811628-8c312900-67f2-11ea-9b72-42e2e0504b37.png)

After the update the new run panel doesn't show anything

**Comment by knopp** [comment]
Can you give me exact vscode version and customize UI version? Also, is there any error in javascript console? (in main menu: Help -> Toggle developer tools and then Developer: Reload window command)

**Comment by lincolnaleixo** [comment]
Sure, 
```
customize UI v: 0.1.34
monkey patch v: 0.1.7
Vscode Version: 1.43.0
Commit: 78a4c91400152c0f27ba4d363eb56d2835f9903a
Date: 2020-03-09T19:34:44.548Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
```

Console error exists indeed:

```
notificationsAlerts.ts:38 TypeError: Cannot read property 'view' of undefined
    at V.replacement (fonts.js:152)
    at V.where.<computed> (utils.js:10)
    at V.render (paneview.ts:202)
    at V.render (viewPaneContainer.ts:249)
    at L.onDidAddViewDescriptors (viewPaneContainer.ts:844)
    at L.create (viewPaneContainer.ts:580)
    at L.create (debugViewlet.ts:85)
    at t.create (panecomposite.ts:42)
    at e.showComposite (compositePart.ts:217)
    at e.doOpenComposite (compositePart.ts:148)
    at e.openComposite (compositePart.ts:112)
    at e.doOpenViewlet (sidebarPart.ts:287)
    at e.openViewlet (sidebarPart.ts:238)
    at H.run (viewlet.ts:172)
    at Object.triggerAndDisposeAction (actions.ts:119)
    at /Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-browser/workbench/async file:/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2791
onDidChangeNotification @ notificationsAlerts.ts:38
(anonymous) @ notificationsAlerts.ts:26
fire @ event.ts:587
addNotification @ notifications.ts:183
error @ notificationService.ts:57
(anonymous) @ actions.ts:96
async function (async)
(anonymous) @ actions.ts:94
invokeFunction @ instantiationService.ts:67
_tryExecuteCommand @ commandService.ts:84
(anonymous) @ commandService.ts:73
Promise.then (async)
executeCommand @ commandService.ts:73
_doDispatch @ abstractKeybindingService.ts:200
_dispatch @ abstractKeybindingService.ts:159
(anonymous) @ keybindingService.ts:236
console.ts:137 [Extension Host] request undefined {command: "loadedSources", type: "request", seq: 16}

```

My settings:

```
  "customizeUI.titleBar": "inline",
  "customizeUI.activityBar": "bottom",
  "customizeUI.listRowHeight": 20,
```

![Screen Shot 2020-03-17 at 11 31 21 AM](https://user-images.githubusercontent.com/2906581/76847751-d4783780-6842-11ea-9142-9da7620c41d0.png)

**Comment by knopp** [comment]
This would be pretty easy to fix if I could reproduce the problem. Unfortunately even with latest version and same configuration I can't, so there must be something else going on.

Can you try disabling monkey patch (Disable Monkey Patch) and send me screenshot of the Run panel?

**Comment by fuchstim** [comment]
I'm having very similar issues, but with the Source Control Panel. 
After starting VS Code, It looks like this:
![image](https://user-images.githubusercontent.com/20535595/87145113-1ffa9980-c2a9-11ea-867b-3d789706f04b.png)

After switching panels and going back to the Source Control Panel, it's completely empty:
![image](https://user-images.githubusercontent.com/20535595/87145162-330d6980-c2a9-11ea-941e-82bc144bf31e.png)

Finally, after reloading VS Code, it works as intended:
![image](https://user-images.githubusercontent.com/20535595/87145308-664ff880-c2a9-11ea-92f0-05f1889a16de.png)

The stacktraces are as follows:
After starting VS Code:
```
ERR Cannot read property 'domNode' of null: TypeError: Cannot read property 'domNode' of null
    at E.updateItemInDOM (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:512:636)
    at E.render (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:510:905)
    at E.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:506:377)
    at O.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:655)
    at Object.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:119:927)
    at t.IndexTreeModel.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:104:685)
    at t.ObjectTreeModel.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:112:728)
    at c.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:118:686)
    at t.CompressibleObjectTreeModel.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:121:776)
    at B.updateElementHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1112:242)
    at e.updateHeight (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7256:98)
    at a (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7229:214)
    at l (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7229:316)
    at e.renderElement (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7229:443)
    at c.renderElement (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1112:937)
    at x.renderElement (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1084:219)
    at W.renderElement (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:541:312)
    at E.insertItemInDOM (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:511:912)
    at E._splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:508:54)
    at E.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:506:699)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:994
    at Array.forEach (<anonymous>)
    at t.CombinedSpliceable.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:981)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:565
    at t.EventBufferer.bufferEvents (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:98:214)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:532)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1097:878)
    at Object.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:119:868)
    at t.IndexTreeModel.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:104:116)
    at t.ObjectTreeModel._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:499)
    at t.ObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:371)
    at c._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:117:159)
    at c.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:116:409)
    at t.CompressibleObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:120:771)
    at B.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1114:255)
    at Oe.refresh (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7241:882)
    at Oe._onDidSpliceRepositories (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7239:1004)
    at Oe.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7240:984)
    at c.fire (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:95:821)
    at Ge.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6387:92)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:328
    at Array.map (<anonymous>)
    at y.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:319)
    at i.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3903:24)
    at e.showComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5753:975)
    at e.doOpenComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:1007)
    at e.openComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:539)
    at e.doOpenViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:900)
    at e.openViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:313)
    at e.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6178:913)
    at file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:11:37
    at e.replacement (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/activity-bar.js:534:23)
    at e.where.<computed> (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:10:30)
    at s.runAction (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:740)
    at s.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:598)
    at e.onClick (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:974:176)
    at Immediate._onImmediate (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:973:727)
    at processImmediate (internal/timers.js:439:21)
```

After switching panels:
```
ERR Cannot read property 'templateData' of null: TypeError: Cannot read property 'templateData' of null
    at E.removeItemFromDOM (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:513:504)
    at E._splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:506:964)
    at E.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:506:699)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:994
    at Array.forEach (<anonymous>)
    at t.CombinedSpliceable.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:981)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:565
    at t.EventBufferer.bufferEvents (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:98:214)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:532)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1097:878)
    at Object.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:119:868)
    at t.IndexTreeModel.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:104:116)
    at t.ObjectTreeModel._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:499)
    at t.ObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:371)
    at c._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:117:159)
    at c.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:116:409)
    at t.CompressibleObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:120:771)
    at B.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1114:255)
    at Oe.refresh (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7241:820)
    at Oe._onDidSpliceRepositories (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7239:1004)
    at Oe.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7241:531)
    at c.fire (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:95:821)
    at Ge.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6387:92)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:328
    at Array.map (<anonymous>)
    at y.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:319)
    at i.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3903:24)
    at e.hideActiveComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5755:399)
    at e.doOpenComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:680)
    at e.openComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:539)
    at e.doOpenViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:900)
    at e.openViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:313)
    at e.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6178:913)
    at file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:11:37
    at e.replacement (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/activity-bar.js:534:23)
    at e.where.<computed> (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:10:30)
    at s.runAction (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:740)
    at s.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:598)
    at e.onClick (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:974:176)
    at Immediate._onImmediate (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:973:727)
    at processImmediate (internal/timers.js:439:21)
log.ts:197   ERR Cannot read property 'domNode' of null: TypeError: Cannot read property 'domNode' of null
    at E.updateItemInDOM (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:512:636)
    at E._splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:507:744)
    at E.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:506:699)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:994
    at Array.forEach (<anonymous>)
    at t.CombinedSpliceable.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:30:981)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:565
    at t.EventBufferer.bufferEvents (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:98:214)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:547:532)
    at O.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1097:878)
    at Object.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:119:868)
    at t.IndexTreeModel.splice (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:104:116)
    at t.ObjectTreeModel._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:499)
    at t.ObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:111:371)
    at c._setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:117:159)
    at c.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:116:409)
    at t.CompressibleObjectTreeModel.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:120:771)
    at B.setChildren (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1114:255)
    at Oe.refresh (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7241:882)
    at Oe._onDidSpliceRepositories (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7239:1004)
    at Oe.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:7240:984)
    at c.fire (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:95:821)
    at Ge.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6387:92)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:328
    at Array.map (<anonymous>)
    at y.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6405:319)
    at i.setVisible (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3903:24)
    at e.showComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5753:975)
    at e.doOpenComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:1007)
    at e.openComposite (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:5751:539)
    at e.doOpenViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:900)
    at e.openViewlet (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6346:313)
    at e.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:6178:913)
    at file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:11:37
    at e.replacement (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/activity-bar.js:534:23)
    at e.where.<computed> (file:///Users/tim/Library/Application Support/Code/User/globalStorage/iocave.customize-ui/modules/utils.js:10:30)
    at s.runAction (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:740)
    at s.run (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:123:598)
    at e.onClick (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:974:176)
    at Immediate._onImmediate (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:973:727)
    at processImmediate (internal/timers.js:439:21)
```

After reloading there are no errors logged in the console.

Versions Numbers:
Customize UI Version: 0.1.45
Monkeypatch Version 0.1.8
VS Code Version: 1.47.0
Commit: d5e9aa0227e057a60c82568bf31c04730dc15dcd
Date: 2020-07-09T08:01:54.115Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.5.0

**Comment by knopp** [comment]
I just published 0.1.46. Let me know if it fixes the problem for you.

**Comment by fuchstim** [comment]
That was quick! I gave it a brief test and it seems to be working fine, I'll update should that change. 
Thanks!
